src/api/containers.rs

```diff
@@ -522,7 +522,14 @@ impl Container {
         self.podman
             .post_json(&ep, Payload::Json(opts.serialize()?), Headers::none())
             .await
-            .map(|resp: models::IdResponse| Exec::new(self.podman.clone(), resp.id))
+            .map(|resp: models::IdResponse| {
+                let is_tty = opts.params.get("tty").and_then(|v| v.as_bool()).unwrap_or_default();
```

Comment:
Everything works fine if the key starts with a capital letter. 
```suggestion
                let is_tty = opts.params.get("Tty").and_then(|v| v.as_bool()).unwrap_or_default();
```

Reply:
Fixed, thanks!